src/tests/unit/logout.test.js

```diff
@@ -0,0 +1,94 @@
+import chai from 'chai';
+import chaiHttp from 'chai-http';
+import { Sequelize } from 'sequelize';
+import app from '../../index';
+import { logout } from '../../controllers/user.controller.js';
+// import User from '../../models/user.model';
+import migration from '../../database/migrations/20230322094623-create-user';
+import redisClient from '../../helpers/redis';
+
+const { expect } = chai;
+chai.use(chaiHttp);
+
+describe('Testing the logout of existing user', function () {
+  let sequelize;
+  let data;
+  before(async function () {
+    sequelize = new Sequelize(process.env.TEST_DATABASE_URL, {
+      dialect: 'postgres',
+      logging: false,
+    });
+    const User = sequelize.define(
+      'User',
+      {
+        id: {
+          type: Sequelize.UUID,
+          defaultValue: Sequelize.UUIDV4,
+          primaryKey: true,
+        },
+        email: {
+          type: Sequelize.STRING,
+          allowNull: false,
+          unique: true,
+        },
+        username: {
+          type: Sequelize.STRING,
+          allowNull: false,
+          unique: true,
+        },
+        password: {
+          type: Sequelize.STRING,
+          allowNull: false,
+        },
+      },
+      { timestamps: true }
+    );
+    await sequelize.sync({ force: true });
+    // Define the users table
+    data = await User.create({
+      email: 'johndoe@example.com',
+      username: 'johndoe',
+      password: 'password123',
+    });
```

Comment:
You should be testing the User model from `src/database/models` directly not creating new model
```suggestion
    
```

Reply:
resolved